src/main/java/com/spotify/docker/client/messages/HostConfig.java

```diff
@@ -1042,6 +1053,15 @@ public Integer oomScoreAdj() {
       return oomScoreAdj;
     }
 
+    public Boolean autoRemove() {
+      return autoRemove;
+    }
+
+    public Builder autoRemove(final Boolean autoRemove) {
```

Comment:
On second thought I am happy to merge this now, in advance of the 1.25 API being released, but would you mind adding a javadoc comment here that this field only has relevance for versions >= 1.25?